packages/network_info_plus/network_info_plus/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.3.0
+- Android: Add `getLocationServiceAuthorization` and `requestLocationServiceAuthorization` permissions support
+
```

Comment:
Please revert this change, check the new CONTRIBUTOR guidelines, you don't have to update the CHANGELOG anymore.